src/utils/accs.js

```diff
@@ -42,11 +42,11 @@ export async function getAndSelectInstances () {
     }
 
     const choices = DEFAULT_TENANTS.concat(
-      resp?.tenants?.map(tenant => `${tenant.name}: ${tenant.instanceURL}/graphql`) || []
+      resp?.tenants?.map(tenant => `${tenant.name}: ${tenant.serviceURLs.graphql}`) || []
     )
 
     const choice = await promptSelect(
-      'Select tenant',
+      'Select tenant (type tenant name to search)',
```

Comment:
@revanth0212 does this allow type-to-search? I don't remember.

If not, we need to make sure we add it as the list will be 100 options :)